architecture/cicd_gitops.adoc

```diff
@@ -0,0 +1,62 @@
+
+Automating both your cluster configuration and container development process by using GitOps allows you to pick and choose where and when to adopt this practice, using {product-title}. Using a CI pipeline that pairs with your GitOps strategy and execution plan is ideal. {product-title} provides the flexibility to choose when and how you integrate this methodology into your business practices and pipelines.
+
+GitOps works well with {product-title} because you can declaratively configure it and the state of the cluster configuration can be stored using Git. For more information, see xref:../installing/install_config/customizations.adoc#customizations[Available cluster customizations].
+
+[id="cicd_gitops_cluster_administration"]
+=== GitOps for single-cluster and multi-cluster administration
+
+Whether you need one or multiple independent or co-operative {product-title} clusters, you can use a GitOps strategy to manage the following tasks:
+
+* Ensure that these clusters have similar states for configuration, monitoring, or storage.
+* Recover or recreate clusters from a known state.
+* Create clusters with a known state.
+* Apply configuration changes to multiple {product-title} clusters.
+* Revert configuration changes to multiple {product-title} clusters.
+* Associate templated configuration with different environments.
+
+[id="cicd_gitops_application_configuration"]
+=== GitOps for application configuration management
+
+You can also use GitOps practices to manage application configuration. This practice ensures consistency in applications when you must deploy them to different clusters in different environments, like development, stage, and production, during their life cycle. Managing application configuration with GitOps is also beneficial when you must deploy applications across multiple clusters, whether on-cloud or on-premise, for availability and scalability purposes.
+
+You can use a GitOps strategy to:
+
+* Promote applications across clusters, from stage to production.
+* Apply application changes to multiple OCP clusters.
+* Revert application changes to multiple OCP clusters.
+
+[id="cicd_gitops_integrators"]
+=== GitOps technology providers and integrators
+
+There are several community offerings and third-party vendors that provide a high level of integration with {product-title}.
+
+You can integrate GitOps into {product-title} with the following community partners and third party integrators:
+
+* ArgoCD
```

Comment:
I see this setup as useful if we plan to introduce additional integrators. But if not, I think it's overboard to have this in a list. Suggest a single sentence:
There are several community offerings and third-party vendors that provide a high level of integration with OpenShift Container Platform, such as ArgoCD.